openshift/kustomize/overlays/openshift/olmv1-ns/patches/manager_deployment_certs.yaml

```diff
@@ -0,0 +1,12 @@
+- op: add
+  path: /spec/template/spec/volumes/-
+  value: {"name":"operator-controller-certs", "secret":{"optional":false,"secretName":"operator-controller-cert"}}
+- op: add
+  path: /spec/template/spec/containers/0/volumeMounts/-
+  value: {"name":"operator-controller-certs", "mountPath":"/var/certs"}
+- op: add
+  path: /spec/template/spec/containers/0/args/-
+  value: "--tls-cert=/var/certs/tls.crt"
```

Comment:
I know there was an issue upstream with `tls.crt` vs. `tls.cert`, and I want to make sure this is working fine. I believe it had to do with the HTTPS client seeing `tls.key` and not finding `tls.cert`, because the CAs and the server certs were in the same directory. Something to look out for, if this fails badly (or strangely).